sdk/communication/Azure.Communication.Chat/tests/ChatClients/ChatClientsTest.cs

```diff
@@ -84,22 +91,74 @@ public void E2E_ThreadCreateUpdateGetDelete_MemberAddUpdateRemove_MessageGetSend
             var threadsCount = threads.Count();
 
             #region Snippet:Azure_Communication_Chat_Tests_E2E_SendMessage
-            SendChatMessageResult sendChatMessageResult = chatThreadClient.SendMessage("This is message 1 content", ChatMessagePriority.High, displayNameMessage);
+            string messageContent = "Content for message 1";
+            SendChatMessageResult sendChatMessageResult = chatThreadClient.SendMessage(messageContent, ChatMessagePriority.High, displayNameMessage);
             #endregion Snippet:Azure_Communication_Chat_Tests_E2E_SendMessage
-            SendChatMessageResult sendChatMessageResult2 = chatThreadClient.SendMessage(messageContent, ChatMessagePriority.High, displayNameMessage);
+            var messageContent2 = "Content for message 2";
```

Comment:
A loop would make this setup more readable

Reply:
I can look at having an array of some sort to keep ids, names and content. Will tackle this in a follow up PR.